lang/cs/Org.Apache.REEF.Network/packages.config

```diff
@@ -18,10 +18,49 @@ specific language governing permissions and limitations
 under the License.
 -->
 <packages>
+  <package id="Microsoft.Extensions.Caching.Abstractions" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Caching.Memory" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Options" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Primitives" version="1.1.0" targetFramework="net451" />
```

Comment:
1) I did put a jira message exactly like you specified above in my checkin, did that not show up?

2) This was a result of converting the System.Runtime.MemoryCache to Microsoft.Extensions.Memory.Caching.MemoryCache which requires 4.5.1 and .net standard 2.0.  Again the goal is to move to .Net Standard 2.0 and this experiment was trying to see if moving to 4.5.1 will help us get to our eventual goal.  If we cant do this then we need to create our own custom caching mechanism to stick with 4.5.  Please advise on a logical plan to get past this